backend/app/models/__init__.py

```diff
@@ -1,18 +1,17 @@
-from sqlalchemy import SQLAlchemy
+import os
 
-db = SQLAlchemy()
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
 
+from .Base import Base
```

Comment:
``
```suggestion

```

Reply:
We need that for `Base.metadata.create_all(bind=engine)`, but I might end up removing that anyway.

Reply:
Note: removing Base.metadata.create_all(bind=engine) so that we rely only on migrations to create tables.